signac/core/h5store.py

```diff
@@ -55,12 +55,22 @@ def _requires_tables():
 logger = logging.getLogger(__name__)
 
 
+class ClosedFileError(RuntimeError):
```

Comment:
Since this is being implemented specifically for `H5Store` and not for signac as a whole, would the name `ClosedH5StoreError` be more appropriate?